src/Builder/Component/Model.php

```diff
@@ -571,6 +498,65 @@ public function build(): void
         }
     }
 
+    /**
+     * @param array $linesCode
+     * @param string $pattern
+     * @param ReflectionProperty|ReflectionClassConstant $attribute
+     *
+     * @return null|string
+     */
+    protected function getAttribute($linesCode, $pattern, $attribute)
+    {
+        $endLine = $startLine = 0;
+        foreach ($linesCode as $line => $code) {
+            if (preg_match($pattern, $code)) {
+                $startLine = $line;
+                break;
+            }
+        }
+        if (!empty($startLine)) {
+            $countLines = count($linesCode);
+            for ($i = $startLine; $i < $countLines; $i++) {
+                if (preg_match('/;(\s*)$/', $linesCode[$i])) {
+                    $endLine = $i;
+                    break;
+                }
+            }
+        }
+
+        if (!empty($startLine) && !empty($endLine)) {
+            $attributeDeclaration = join(
+                '',
+                array_slice(
+                    $linesCode,
+                    $startLine,
+                    $endLine - $startLine + 1
+                )
+            );
+            $attributeFormatted = $attributeDeclaration;
+            if (!empty($attribute->getDocComment())) {
+                $attributeFormatted = "    " . $attribute->getDocComment() . PHP_EOL . $attribute;
+            }
+            return $attributeFormatted;
+        }
+
+        return null;
+    }
+
+    /**
+     * @param string $fieldName
+     *
+     * @return string
+     */
+    protected function getFieldName($fieldName)
+    {
+        if ($this->modelOptions->getOption('camelize')) {
```

Comment:
```suggestion
        if ($this->modelOptions->hasOption('camelize')) {
```

Probable `hasOption` is more correct here?

Reply:
`camelize` is set during initialization to boolean value (`true` if set via command or `false` by default).
`hasOption` will return always true is this case

Reply:
More correct check could be
```
if ($this->modelOptions->hasOption('camelize') && $this->modelOptions->getOption('camelize')) {
```
But seems overhead

Reply:
Ok